Libs/Widgets/ctkFileDialog.cpp

```diff
@@ -145,6 +142,10 @@ ctkFileDialog::~ctkFileDialog()
 //------------------------------------------------------------------------------
 void ctkFileDialog::setBottomWidget(QWidget* widget, const QString& label)
 {
+  if (this->Options)
```

Comment:
```suggestion
if (!(q->Options & DontUseNativeDialog))
```

Reply:
With that change:

> Error	C2065	'q': undeclared identifier	CTKWidgets	C:\S5R-nightly\CTK\Libs\Widgets\ctkFileDialog.cpp	145

Reply:
I haven't compiled the code. You need to use the scope wherever Options is defined in(d->, q->, this->...).